src/modules/RA/DataSource/components/DataSourceCSVFields.js

```diff
@@ -179,7 +179,10 @@ const DataSourceCSVFields = ({ translate: t, type, ...props }) => (
       step={1}
     />
 
-    <h4>{t('datasource.form.optionsCSV.options')}</h4>
+    <>
+      <h4>{t('datasource.form.optionsCSV.options')}</h4>
+    </>
```

Comment:
I don't see the utility of this diff, why did you add a fragment tag?

Reply:
To protect this element from react-admin props I assume ?

Reply:
> To protect this element from react-admin props I assume ?

Precisely.

*react-admin component do not "just" render its children components. It iterate through them and provides each some props. Many of these props aren't valid for DOM elements. Wrapping with a Fragment make it the receiver of the props, keeping safe the DOM element.*